### PR TITLE
Porting query_execution test suite from legacy test schedule.

### PR DESCRIPTION
**Pull request by foyzur**
Signed-off-by: George Caragea <gcaragea@pivotal.io>

### CODE REVIEW

File: src/test/regress/sql/qp_query_execution.sql [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+select qx_count_operator('explain select foo_p.b, foo_p.t from foo_p left outer join bar_p on foo_p.a = bar_p.a  where foo_p.t is not null and foo_p.a = 6;', 'Hash Left Join');
+-- end_ignore
+select foo_p.b, foo_p.t from foo_p left outer join bar_p on foo_p.a = bar_p.a  where foo_p.t is not null and foo_p.a = 6 order by 1, 2 asc limit 10;
+
+-- Test Rail: C87790 (Motion nodes on top of other non-projection capable nodes)
+-- 
+
+-- Test Rail: C87849 (Queries with inner join)
+-- TBD
+
```
Comment (vraghavan78) [inline src/test/regress/sql/qp_query_execution.sql]: remove useless lines

File: src/test/regress/sql/qp_query_execution.sql [hunk 2]
```diff
@@ -0,0 +1,719 @@
...
+select qx_count_operator('explain select foo_p.b, foo_p.t from foo_p inner join bar_p on foo_p.a = bar_p.a  where foo_p.t is not null and foo_p.a = 6;', 'Nested Loop'); 
+select foo_p.b, foo_p.t from foo_p inner join bar_p on foo_p.a = bar_p.a  where foo_p.t is not null and foo_p.a = 6 order by 1, 2 desc limit 10;
+
+-- Test Rail: C87790 (Motion nodes on top of other non-projection capable nodes)
+-- TBD
+
+-- Test Rail: C87849 (Queries with inner join)
+-- TBD
+
```
Comment (vraghavan78) [inline src/test/regress/sql/qp_query_execution.sql]: same here

**Comment by vraghavan78** [comment]
+1

**Comment by armenatzoglou** [comment]
LGTM

**Comment by hsyuan** [comment]
+1

**Comment by danielgustafsson** [comment]
I'm not a fan of naming test tables foo and bar and then create/drop to reuse the foo/bar names, a bit more descriptive names are appreciated when the test in question actually fails. That might be for a followup PR since I guess this is more about porting the tests over vanilla. I find it important that we actually get to cleaning these up though since we are amassing quite the backlog of things to clean with these PRs.

Also, I agree that creating a complex table like `orders_test` belongs in a test case it seems that a query execution test suite could do something more exciting than just create/drop? At the very least these should a comment explaining why a query execution suite seemingly doesn't execute a query IMO.

**Comment by foyzur** [comment]
Pushed to master: 9d19664ccbe2b81069249924538f61bba8179b8d and 29a0fec8fe4c45c0730f81bd91bc5fc0b9c772bb

**Pull request closed without merging**